eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -162,8 +165,7 @@ class Router(val nodeParams: NodeParams, watcher: ActorRef, initialized: Option[
 
   setTimer(TickBroadcast.toString, TickBroadcast, nodeParams.routerConf.routerBroadcastInterval, repeat = true)
   setTimer(TickPruneStaleChannels.toString, TickPruneStaleChannels, 1 hour, repeat = true)
-
-  val SHORTID_WINDOW = 100
+  setTimer(TickNetworkStats.toString, TickNetworkStats, 6 hour, repeat = true)
```

Comment:
```suggestion
  setTimer(TickNetworkStats.toString, TickNetworkStats, 6 hours, repeat = true)
```

Reply:
Should that be configurable?

Reply:
It would be a good idea, yeah. I was lazy for that first iteration :)